applications/app/views/survey/emailDigest404.scala.html

```diff
@@ -0,0 +1,45 @@
+@(surveyPage: Page)(implicit request: RequestHeader)
+
+@import model.Page
+@import conf.Configuration.Survey.formStackAccountName
```

Comment:
Spoke with @janua - this isn't necessary for now, we can revise how we do these things later when we potentially separate it into another app